lib/sles4sap/sap_deployment_automation_framework/configure_tfvars.pm

```diff
@@ -14,6 +14,8 @@ use Carp qw(croak);
 use utils qw(file_content_replace);
 use sles4sap::sap_deployment_automation_framework::deployment qw(get_os_variable);
 use sles4sap::sap_deployment_automation_framework::deployment_connector qw(find_deployment_id);
+use Mojo::Base qw(publiccloud::basetest publiccloud::azure);
```

Comment:
Is it possible to keep this dependency at testmodule level?

Reply:
Sure, let me try.